### PR TITLE
Fix rdm_cntr_pingpong, add runfabtests.sh cluster mode

### PR DESCRIPTION
**Pull request by ztiffany**
@hppritcha @sungeunchoi @jshimek @jswaro

### CODE REVIEW

File: pingpong/rdm_cntr_pingpong.c [hunk 1]
```diff
@@ -91,18 +91,18 @@ static int recv_xfer(int size)
...
 	if (ret)
 		FT_PRINTERR("fi_recv", ret);
 	recv_outs++;
 
+	ret = fi_cntr_wait(rxcntr, recv_outs, -1);
+	if (ret < 0) {
+		FT_PRINTERR("fi_cntr_wait", ret);
+		return ret;
+	}
+
```
Comment (hppritcha) [inline pingpong/rdm_cntr_pingpong.c:105]: why is it necessary to move the cntr_wait till after the fi_recv?  Is there something different about the GNI provider that causes the client to not post a final receive some times?

**Comment by hppritcha** [comment]
This looks good to me although I'd like to understand better the change to the test.  Also looks like
this PR needs to be rebased.

File: pingpong/rdm_cntr_pingpong.c (hunk 1 above)
Comment (ztiffany) [inline pingpong/rdm_cntr_pingpong.c:105]: fi_cntr_wait() is being used for completions.  Due to the ordering (cntr_wait() then fi_recv()).  the old code effectively doesn't wait for a completion for the final recv.  When the client exits before completing it's final receive, the send side can't complete a rendezvous send.  The test can't hang when using eager path message sizes in GNI.  I assume other providers don't rely on receive side progress like we do with rendezvous.

**Comment by hppritcha** [comment]
:+1: modulo the need to rebase

File: pingpong/rdm_cntr_pingpong.c (hunk 1 above)
Comment (sungeunchoi) [inline pingpong/rdm_cntr_pingpong.c:105]: Is this something that should be pushed upstream?

**Comment by hppritcha** [comment]
do we still want this to go in to fabtests?

**Comment by sungeunchoi** [comment]
Closing this since a lot has changed, but it should be a reference for #67.

**Pull request closed without merging**